mkdocs.yml

```diff
@@ -27,6 +27,7 @@ nav:
       - "Chapter 6: Orchestrate the workflow with a CI/CD pipeline": the-guide/chapter-6-orchestrate-the-workflow-with-a-cicd-pipeline/index.md
       - "Chapter 7: Track model evolutions in the CI/CD pipeline with CML": the-guide/chapter-7-track-model-evolutions-in-the-cicd-pipeline-with-cml/index.md
       - "Chapter 8: Serve the model with MLEM": the-guide/chapter-8-serve-the-model-with-mlem/index.md
+      - "Chapter 9: Cleaning up resources and environments": the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md
```

Comment:
As some of the cleanup is already in the "Conclusion" chapter, what if we renamed the chapter "Conclusion" to "Clean up and Conclusion" and merge these chapters together? What do you think?

Reply:
It makes much more sense to merge both.